### ISSUE TITLE
SoundcoreDesktop app does not lounch

### ISSUE THREAD

**Issue by Usernames-father** [KillerBOSS2019/SoundcoreLifeAPI#4]
I'm operating windows 10 and the app failes to lounch
![Screenshot_4](https://user-images.githubusercontent.com/106969700/176173555-ac45d79f-f1bb-451e-add4-5a038dbe35e4.png)

**Comment by DamienStaebler** [comment]
Have you tried running the Python script itself?

**Comment by Usernames-father** [comment]
> Have you tried running the Python script itself?

No, could you please tell me how to do that?

**Comment by Usernames-father** [comment]
![image](https://user-images.githubusercontent.com/106969700/187155220-a94ac6b9-68d6-4083-a65d-f2149fc6e823.png)
I've unpacked the base_library zip inside the main folder, it's the only way to make any window appear and only when running as administrator

**Comment by DamienStaebler** [comment]
1. Is your phone disconnected from the headphones?

2. Have your headphones been connected to a mobile app before? If not it needs to connect at least once.

3. What model is your headphone?

**Comment by Usernames-father** [comment]
Yes. The problem was the app that couldn't launch at all and shoved errors
above. The solution was to unpack base archive inside the main folder and
to turn of the "read only" in folder's property!
Thank you for your answers and for the app as well!)

пн, 29 авг. 2022 г., 23:08 IMISSED ***@***.***>:

**Comment by DamienStaebler** [comment]
Nice! Glad it works

**Comment by Usernames-father** [comment]
No, i haven't. I don't really understand what python script will do on
windows 10.

сб, 23 июл. 2022 г., 01:00 IMISSED ***@***.***>:

**Comment by DamienStaebler** [comment]
I'll probably create another version of this application, I did not expect to get this much attention lol

**Comment by Usernames-father** [comment]
Glad to hear it!

**Comment by ricardogroto** [comment]
I'm using Windows 11, I followed your step by step and the app doesn't even start. Can you help me? @KillerBOSS2019

**Comment by ricardogroto** [comment]
I'm using Windows 11, I followed your step by step and the app doesn't even start. I installed Python 3, I also installed Tkinter. I tried to correct any possible problem like the people mentioned above, and it simply does not open Soundcore Desktop. 
Can you help me?

**Comment by ricardogroto** [comment]
I'm using Windows 11, I followed your step by step and the app doesn't even start. I installed Python 3, I also installed Tkinter. I tried to correct any possible problem like the people mentioned above, and it simply does not open Soundcore Desktop.
Can you help me? @KillerBOSS2019

**Comment by DamienStaebler** [comment]
https://github.com/KillerBOSS2019/SoundcoreDesktop/releases/tag/1.0

Download the zip and unzip it. There should be an exe file

**Comment by ricardogroto** [comment]
> https://github.com/KillerBOSS2019/SoundcoreDesktop/releases/tag/1.0
> 
> Download the zip and unzip it. There should be an exe file

I just redo the process all over again. When I click on the exe, it just doesn't open, nothing happens. I would love to use the Soundcore app you modified to use on Windows. I also reinstalled Python 3 to its latest version 3.11.4. I also reinstalled Tkinter using pimp from cmd.

**Comment by DamienStaebler** [comment]
that's odd. u shouldn't need python and stuff. since it's already included with the zip package. does it give any error at all?

**Comment by ricardogroto** [comment]
> isso é estranho. você não deveria precisar de python e outras coisas. uma vez que já está incluído no pacote zip. dá algum erro em tudo?

Absolutely no errors happen. When I click to run the SoundcoreDesktop exe, nothing happens, it doesn't give an error, and nothing opens either.

**Comment by DamienStaebler** [comment]
I'll have to take a look when I have time.

**Comment by Usernames-father** [comment]
![image](https://github.com/KillerBOSS2019/SoundcoreDesktop/assets/106969700/ef60988a-24fd-4d0c-9ef8-b57e1c6d7491)
Hello! So my new system can not launch the application with this error. Could you please help? The previous steps made no difference.

**Comment by DamienStaebler** [comment]
In that folder do you see a file named "lastconnect.json"? Delete that

**Comment by Usernames-father** [comment]
![image](https://github.com/KillerBOSS2019/SoundcoreDesktop/assets/106969700/6b659152-8864-4303-a3b3-ea0aff7efd8e)
Yeah, I have deleted that one and now I have new old error)

**Comment by DamienStaebler** [comment]
When does it create a lastconnect.json? If so what's the content of it?

**Comment by Usernames-father** [comment]
[lastConnect.json](https://github.com/KillerBOSS2019/SoundcoreDesktop/files/13177526/lastConnect.json)
I do not know how to open it for sharing the content, so here is this file. It appears after turning off a read-only option of files and trying to launch the app via SoundcoreDesktop.exe. After the removal of that file, I see the error above and after a while appears this message
![image](https://github.com/KillerBOSS2019/SoundcoreDesktop/assets/106969700/5f986c35-7058-4c76-a915-d8696b50778e)

**Comment by Usernames-father** [comment]
The last address is my q45 headphones, as i discovered, so I removed them from Bluetooth devices and now the macaddres error appears again

**Comment by DamienStaebler** [comment]
You can open the lastconnect.json with any text editor. One way to solve this is to manually enter your Mac address which can be found using this method https://support.litheaudio.com/portal/en/kb/articles/how-to-find-mac-address-of-bluetooth-speaker once you find your Mac address enter it in lastconnect.json. also remember this program is made specifically for Q30 and some people say it works on Q35

**Comment by DamienStaebler** [comment]
And I do not own Q45 headphones therefore I do not know how to mimic the communication

**Comment by angelrojo91** [comment]
I want to thank you for sharing this great work that now allows me to listen to music, play and work with headphones. It works perfectly. I had to disconnect the headphones from the phone first.

**Comment by DamienStaebler** [comment]
I am glad you found it helpful (:

**Comment by ricardogroto** [comment]
Thanks mate, is finally working here. You definitely did an amazing job here! 

![Image](https://github.com/user-attachments/assets/954365e9-d969-4781-98f3-d32ad1f5ad3f)
